src/main/app/documents/documentsClient.ts

```diff
@@ -28,4 +28,21 @@ export class DocumentsClient {
     })
   }
 
+  getClaimIssueCopy (claimExternalId: string): http.IncomingMessage {
+    if (StringUtils.isBlank(claimExternalId)) {
+      throw new Error('Claim external ID cannot be blank')
+    }
+    return requestNonPromise.get({
+      uri: `${this.documentsUrl}/claimIssueReceipt/${claimExternalId}`
+    })
+  }
+  getDefendantResponseReceipt (claimExternalId: string): http.IncomingMessage {
+    if (StringUtils.isBlank(claimExternalId)) {
+      throw new Error('Claim external ID cannot be blank')
+    }
+    return requestNonPromise.get({
```

Comment:
All methods retrieving documents are the same, only difference is in part of the path indicating document type, we should have private, parametrised method that just takes the document type and makes HTTP call.

Reply:
Done